### PR TITLE
(#78) Added Search component

### LINKED ISSUE #78: Create Search component
*Create `Search` component.*

**Parent Issue:** #76

### Sub-Tasks
- [x] Add "Starts with" radio button (update component test)
- [x] Add "Contains" radio button (update component test)
- [x] Add searchbox and search button (update component test)
- [x] Add styling
- [x] Implement search button functionality to use provided form inputs to direct user to search route  (update component test)
- [x] Refactored `TextInput` component and added tests for `TextInput`, `Radio`, and `InputLabel` components

### PR DESCRIPTION
**Pull request by seyilonge-nci**
Closes #78 
Increased test coverage

Removed unwanted file

Put back missing required prop removed in error

### CODE REVIEW

File: src/components/search/Search.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import { useParams, useNavigate } from 'react-router';
+
+import './Search.scss';
+
+import Radio from '../atomic/Radio';
+import TextInput from '../atomic/TextInput';
+import { testIds } from '../../constants';
+
+const Search = () => {
+    const containsText = 'contains';
```
[the remark this answers is not in the document]
Comment (seyilonge-nci) [reply src/components/search/Search.jsx]: Updated variable declaration to correctly reflect use. Added support for Spanish text.

File: src/components/search-box/SearchBox.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './SearchBox.scss';
+
+import Search from '../search';
+
+const SearchBox = () => {
+    return (
+        <div className="search-box-container">
```
Comment (arcepaul) [inline src/components/search-box/SearchBox.jsx]: Shouldn't this be a `<form>`?

File: src/components/search-box/SearchBox.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import './SearchBox.scss';
```
Comment (arcepaul) [inline src/components/search-box/SearchBox.jsx]: this should be pulled in via dictionaries.scss

File: src/components/atomic/TextInput/TextInput.jsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+
+import './TextInput.scss';
```
Comment (arcepaul) [inline src/components/atomic/TextInput/TextInput.jsx]: should be pulled in via dictionaries.scss

File: src/components/search-box/SearchBox.jsx (hunk 2 above)
Comment (seyilonge-nci) [reply src/components/search-box/SearchBox.jsx]: Fixed and added `aria-label` to conform with prod.

File: src/components/search-box/SearchBox.jsx (hunk 3 above)
Comment (seyilonge-nci) [reply src/components/search-box/SearchBox.jsx]: Done

File: src/components/atomic/TextInput/TextInput.jsx (hunk 4 above)
Comment (seyilonge-nci) [reply src/components/atomic/TextInput/TextInput.jsx]: Done

**Pull request closed without merging**